### PR TITLE
Add layout option to com_contact category menu item

### PR DESCRIPTION
**Pull request by danielmorell**
Added `contact_layout` field to the com_contact category view.

This allows users to select the layout for the contact view that is for the com_contact category menu item.

Pull Request for Issue # .

### Summary of Changes

I added a single field to the com_contact `category` view. It allows users to select the layout that is used for the `contact` view. 

### Testing Instructions

1. Setup new Joomla installation.
1. Create template override for com_contact `contact` view.
1. Create a contact category.
1. Create a contact in the new category.
1. Create a contact category menu item.
1. Note missing field.

Verify the change...
1. Implement the code from this PR.
1. Note the field is not missing anymore.
1. Select the template override created in step 1, and save the menu item.
1. Note override being used on the front-end contact view for the menu item.

### Expected result

There should be a "Choose a Layout" field in the "Contact Display Options" tab.

### Actual result

The `presentation_style` field is present but not the "Choose a Layout" field.

This result can be achieved by adding the field to the category view .xml file. 

### Documentation Changes Required

I could not find any documentation on the com_contact category menu item.

### CODE REVIEW

File: components/com_contact/views/category/tmpl/default.xml [hunk 1]
```diff
@@ -327,6 +327,16 @@
 
 		<fieldset name="contact" label="COM_CONTACT_BASIC_OPTIONS_FIELDSET_LABEL"
 			addfieldpath="/administrator/components/com_fields/models/fields">
+			
+			<field
+				name="contact_layout" type="componentlayout"
```
Comment (Quy) [inline components/com_contact/views/category/tmpl/default.xml]:
```suggestion
				name="contact_layout"
				type="componentlayout"
```

Comment (danielmorell) [reply components/com_contact/views/category/tmpl/default.xml]: Thanks, @Quy.

**Comment by viocassel** [comment]
I have tested this item :white_check_mark: successfully on 3cba77058e0e468649cb8d4f12518da7b5e0f094<hr /><sub>This comment was created with the <a href="https://github.com/joomla/jissues">J!Tracker Application</a> at <a href="https://issues.joomla.org/tracker/joomla-cms/23514">issues.joomla.org/tracker/joomla-cms/23514</a>.</sub>

**Comment by ghost** [comment]
I have tested this item :white_check_mark: successfully on 2e01b92a8189526916cd666d468787435929eea8<br /><br />After PR is applied a `Choose a Layout`-Field is shown.<hr /><sub>This comment was created with the <a href="https://github.com/joomla/jissues">J!Tracker Application</a> at <a href="https://issues.joomla.org/tracker/joomla-cms/23514">issues.joomla.org/tracker/joomla-cms/23514</a>.</sub>

**Comment by ghost** [comment]
Status "Ready To Commit".

**Comment by alikon** [comment]
I have tested this item :white_check_mark: successfully on 2e01b92a8189526916cd666d468787435929eea8<hr /><sub>This comment was created with the <a href="https://github.com/joomla/jissues">J!Tracker Application</a> at <a href="https://issues.joomla.org/tracker/joomla-cms/23514">issues.joomla.org/tracker/joomla-cms/23514</a>.</sub>

**Comment by alikon** [comment]
re-invoking the bot

Status "Ready To Commit".<hr /><sub>This comment was created with the <a href="https://github.com/joomla/jissues">J!Tracker Application</a> at <a href="https://issues.joomla.org/tracker/joomla-cms/23514">issues.joomla.org/tracker/joomla-cms/23514</a>.</sub>

**Comment by HLeithner** [comment]
thx

**Pull request merged**